modules/extensions/src/util/comlink.ts

```diff
@@ -1,12 +1,16 @@
 import * as Comlink from "comlink";
-import { ExtensionPortAPI } from "../types";
+import { ExperimentalAPIs, ExtensionPortAPI } from "../types";
 
 export const extensionPort = (() =>
   typeof window !== "undefined"
     ? (Comlink.wrap<ExtensionPortAPI>(
         Comlink.windowEndpoint(self.parent, self, "*")
-      ) as Comlink.Remote<ExtensionPortAPI>)
-    : null)() as Comlink.Remote<ExtensionPortAPI>;
+      ) as Comlink.Remote<ExtensionPortAPI> & {
+        experimental: Comlink.Remote<ExperimentalAPIs>;
+      })
+    : null)() as Comlink.Remote<ExtensionPortAPI> & {
+  experimental: Comlink.Remote<ExperimentalAPIs>;
```

Comment:
what errors do you see without this?

Reply:
Typescript errors.  Says I need to do `await (await extensionPort.experimental).exec(...)`, but that is incorrect.

Reply:
hmm okay, i'll take a shot at something sweet and simple